src/TextLink/index.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import * as React from "react";
+import { darken } from "polished";
+import * as tokens from "orbit-design-token";
+
+type Props = {
+  title: string,
+  url: string,
+  onClick: (e: SyntheticEvent<HTMLButtonElement>) => void,
+  newTab: boolean,
```

Comment:
`type`, `size` and `newTab` should be declared as optional props

Reply:
Changing `type` and `size` will results in flow error `Computed property cannot be accessed with undefined` because these props are used as keys, e.g. `${fontSizeTextLink[size]}`

Reply:
@jaroslav-kubicek they should not, see https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components

Reply:
Leaving as it is.